app/javascript/src/Users/components/FeaturesFieldset.jsx

```diff
@@ -40,6 +45,21 @@ const FeaturesFieldset = ({ features, selectedSections = [], areServicesVisible
   )
 }
 
+/**
+ * A list describing member permissions for each label.
+ * @param {Array} descriptionItems - An array of strings containing the description of the label.
+ */
+const LabelDescriptionItems = ({ descriptionItems }: { descriptionItems: Array<string> }) => {
+  return (
+    <ul className="FeatureAccessList-item--labelDescription">
+      {descriptionItems.map(item => (<li key={ item }>
+        {ReactHtmlParser(item)}
+      </li>
+      ))}
```

Comment:
```suggestion
      {descriptionItems.map(item => <li key={item}>{ReactHtmlParser(item)}</li>)}
```

Reply:
@josemigallas you should finish in line 57